### PR TITLE
Implement simpleComparison() and create load tests for issue #119 and #174

### PR DESCRIPTION
**Pull request by Hannnnnnn404**
# PR Details

- Provide another kind of pattern ```simpleComparison()```, which makes it faster than ```compareImages()```.
- Creare load tests to see if ```simpleComparison()``` method is faster.
## Description

- Implement a method called  ```simpleComparison()```, which makes it faster to compare images without drawing rectangles. 
- Add ```isFirstDifference()``` method to judge the first different pixel of two images.
- Create load tests to verify time differences between ```simpleComparison()``` and ```compareImages()```.
## Related Issue

- issue #119 
- issue #174 
## Motivation and Context

The change is aim to users who don't want to know what differences of compare images. Some users only need to know that if two images are different or not. This ```simpleComparison``` pattern can makes it faster for users to get result quickly.

## How Has This Been Tested

- Add a new test file  called _LoadTest.java_, in which I designed some load test to test speed differernces.
- Verify the ```simpleComparison()``` feature to make sure it could run normally. Check out the compare result ```ComparisonState```.
- For some image instances, calculate running time of ```simpleComparison()``` and ```compareImages()``` method. ```simpleComparison()``` should be faster than ```compareImages()```.
## Types of changes

- [x] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/test/java/com/github/romankh3/image/comparison/LoadTest.java [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        //Check the result
+        assertEquals(ImageComparisonState.MISMATCH, imageComparisonResult2.getImageComparisonState());
+        //check the load differences between simpleComparison and compareImage pattern
+        assertTrue(simpleComparisonTime < compareImageTime);
+    }
+
+    @Test
+    @DisplayName("compare load differences between {@link simpleComparison} method and {@link compareImage} method")
+    public void loadTest2(){
+        //load images to be compared:
```
Comment (romankh3) [inline src/test/java/com/github/romankh3/image/comparison/LoadTest.java:62]: Please, follow general way for creating tests with BDD blocks:
//given
//when
//then

Comment (Hannnnnnn404) [reply src/test/java/com/github/romankh3/image/comparison/LoadTest.java:62]: Do you mean that I should code tests following the BDD syle?

Comment (Hannnnnnn404) [reply src/test/java/com/github/romankh3/image/comparison/LoadTest.java:62]: > Please, follow general way for creating tests with BDD blocks: //given //when //then
Do you mean that I should code tests following the BDD style? And my logic of load tests is right.

**Comment by romankh3** [comment]
thanks for your contribution

**Pull request merged**